dipy/segment/tests/test_clustering.py

```diff
@@ -208,18 +215,20 @@ def test_cluster_centroid_getitem():
     centroid = np.zeros(features_shape)
     cluster = ClusterCentroid(centroid)
     for idx in indices:
-        cluster.assign(idx, (idx+1)*features)
+        cluster.assign(idx, (idx + 1) * features)
 
     # Test indexing
     for i in advanced_indices:
         assert_equal(cluster[i], indices[i])
 
     # Test advanced indexing
-    assert_array_equal(cluster[advanced_indices], [indices[i] for i in advanced_indices])
+    assert_array_equal(
+        cluster[advanced_indices], [
+            indices[i] for i in advanced_indices])
```

Comment:
This is too many lines, and breaking after a `[` doesn't help readability (IMHO). How about the following? 

assert_array_equal(cluster[advanced_indices],
[indices[i] for i in advanced_indices])

With the second line properly indented.